index.html

```diff
@@ -49,5 +49,178 @@ <h2>I'm a Software Developer</h2>
+                <div class="work-img-box">
+                    <img src="images/pro-img-2.png" alt="Profesional Art Printing Data screenshot">
+                </div>
+                <div class="work-content">
+                    <h3 class="work-title">Profesional Art Printing Data</h3>
+                    <p class="work-description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required. has been the industry's standard.
+                    </p>
+                    <ul class="work-languages-list">
+                        <li class="work-language">HTML</li>
+                        <li class="work-language">Bootstrap</li>
+                        <li class="work-language">Ruby</li>
+                    </ul>
+                    <a href="#" class="btn see-work">See project</a>
+                </div>
+            </div>
+            <!-- WORK BOX (end) -->
+            <!-- WORK BOX (begin) -->
+            <div class="work-box">
+                <div class="work-img-box">
+                    <img src="images/pro-img-2.png" alt="Profesional Art Printing Data screenshot">
+                </div>
+                <div class="work-content">
+                    <h3 class="work-title">Profesional Art Printing Data</h3>
+                    <p class="work-description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required. has been the industry's standard.
+                    </p>
+                    <ul class="work-languages-list">
+                        <li class="work-language">HTML</li>
+                        <li class="work-language">Bootstrap</li>
+                        <li class="work-language">Ruby</li>
+                    </ul>
+                    <a href="#" class="btn see-work">See project</a>
+                </div>
+            </div>
+            <!-- WORK BOX (end) -->
+            <!-- WORK BOX (begin) -->
+            <div class="work-box">
+                <div class="work-img-box">
+                    <img src="images/pro-img-2.png" alt="Profesional Art Printing Data screenshot">
+                </div>
+                <div class="work-content">
+                    <h3 class="work-title">Profesional Art Printing Data</h3>
+                    <p class="work-description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required. has been the industry's standard.
+                    </p>
+                    <ul class="work-languages-list">
+                        <li class="work-language">HTML</li>
+                        <li class="work-language">Bootstrap</li>
+                        <li class="work-language">Ruby</li>
+                    </ul>
+                    <a href="#" class="btn see-work">See project</a>
+                </div>
+            </div>
```

Comment:
According to the Figma template, you should follow the design as it is. Kindly I suggest you follow this [link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=1WZiiIvrjUBtySdQ-0) to make your design match the Requirement. 😄 

>### The Figma design
>
><img width="241" alt="Screenshot 2023-04-28 at 14 41 17" src="https://user-images.githubusercontent.com/91237525/235228673-38cc3f86-9ca6-4b01-88a2-2ddb62843e25.png">
>
>
>### Your design
>
><img width="270" alt="Screenshot 2023-04-28 at 14 41 42" src="https://user-images.githubusercontent.com/91237525/235228887-e67d4623-bec6-40a9-81ae-57487b8e1944.png">

Reply:
Also, after analyzing the template, I thought that the boxes should have a normal display and only changed when you hover the mouse or clicked on it.
Thank you @stanleySimeon  for the review.  I will rebuild this part